packages/layout/src/index.js

```diff
@@ -37,7 +37,7 @@ export const breakpoints = {
   sm: {
     width: rem(320),
     columns: 4,
-    margin: 0,
+    margin: rem(0),
```

Comment:
```suggestion
    margin: '0',
```

I don't think `rem` is smart enough to know it's a unitless value yet 😓

Reply:
Ah right, good point.